charts/devtron/templates/NOTES.txt

```diff
@@ -7,4 +7,23 @@
 2. You can watch the progress of Devtron microservices installation by the following command
 
    kubectl -n devtroncd get installers installer-devtron -o jsonpath='{.status.sync.status}'
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+3. You can access devtron dashboard with following.
+{{- if $.Values.components.devtron.ingress.enabled }}
+
+  {{ printf $.Values.components.devtron.ingress.host }}
+{{- else if eq $.Values.components.devtron.service.type "LoadBalancer" }}
+
+  kubectl get svc -n devtroncd devtron-service -o jsonpath='{.status.loadBalancer.ingress}'
+{{- else if eq $.Values.components.devtron.service.type "NodePort" }}
+
+  export PUBLIC_IP=$(kubectl get nodes -o jsonpath="{.items[0].status.addresses[2].address}")
+  
+  Your devtron dashboard is available on http://$PUBLIC_IP:{{ .Values.components.devtron.service.nodePort }}
```

Comment:
like here you can directly add
3. Your devtron dashboard is available on http://$PUBLIC_IP:{{ .Values.components.devtron.service.nodePort }}
also it's not necessary that you'll get nodeport in the values, user may or may not provide, so better get nodeport from service using command